src/main/java/com/todolist/app/servlet/ToDoServlet.java

```diff
@@ -1,21 +1,18 @@
 package com.todolist.app.servlet;
+
+
 import jakarta.servlet.*;
 import jakarta.servlet.http.*;
 import jakarta.servlet.annotation.*;
 import java.io.IOException;
-import java.io.PrintWriter;
-
-
-
 
 
-@WebServlet(name = "ToDoServlet", value = "/hello")
+@WebServlet(name = "ToDoServlet", value = "/notification/")
```

Comment:
The last / can be omited. I mean `value = "/notification/"` to `value = "/notification"`

Reply:
@aholtvianskyi It doesn't work without the last "/"

Reply:
You have to update this line in NewTaskServlet:doPost
```
response.sendRedirect(request.getContextPath() + "/notification/?operation=ADD&id=" + uuid);
```